test/cbmc/proofs/setDataInterface/README.md

```diff
@@ -0,0 +1,20 @@
+setDataaInterface proof
```

Comment:
```suggestion
setDataInterface proof
```

Reply:
Removed it in [00c584f](https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/245/commits/00c584ff56af2ffd29297de27a527a4687706687)